docs/reference/mapping/types.asciidoc

```diff
@@ -22,6 +22,7 @@ string::         <<text,`text`>>, <<keyword,`keyword`>> and <<wildcard,`wildcard
 <<nested>>::    `nested` for arrays of JSON objects
 
 [float]
+[[_spatial_datatypes]]
```

Comment:
The leading `_` is usually an indicator that this was an automatically generated anchor id. Unless you are replacing one that is auto-generated I'd avoid it.

Reply:
👍  will remove the underscore. Note that 'spatial' naming is based on this PR, where I propose grouping all the spatial types into one section: https://github.com/elastic/elasticsearch/pull/59731.